### PR TITLE
[Add] [10.0] hr_employee_birth_name

### PR DESCRIPTION
**Pull request by leemannd**
Add new field in employee's personal informations

### CODE REVIEW

File: hr_employee_birth_name/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (feketemihai) [inline hr_employee_birth_name/README.rst:1]: @leemannd Thanks for your contribution, can you change the readme file, it remained from social media module.

File: hr_employee_birth_name/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# ©  2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Employee Birth Name',
+    'version': '10.0.1.0.0',
+    'category': 'Human Resources',
+    'author': "Camptocamp SA,Odoo Community Association (OCA)",
+    'website': 'http://www.savoirfairelinux.com',
```
Comment (feketemihai) [inline hr_employee_birth_name/__manifest__.py]: @leemannd Can you check the website...no Savoirfairelinux in the contribution or copyright...

**Comment by leemannd** [comment]
@feketemihai Hello, I'll gladly fix the PR. Did you forget to put a comment?

**Comment by feketemihai** [comment]
@leemannd No i have not forgot...see the comment for the readme file and the website in the manifest...

**Comment by leemannd** [comment]
@feketemihai Thanks for the comment/review. Updated

File: hr_employee_birth_name/README.rst (hunk 1 above)
Comment (leemannd) [reply hr_employee_birth_name/README.rst:1]: Changed

**Review by feketemihai — APPROVED** [review]

File: hr_employee_birth_name/README.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+This module adds the following to employees :
+
+* hr_employee_birth_name
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
```
Comment (yvaucher) [inline hr_employee_birth_name/README.rst]: Remove the link on feedback (it was removed from template)

File: hr_employee_birth_name/models/hr_employee.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# copyright  2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    hr_employee_birth_name = fields.Char(
```
Comment (yvaucher) [inline hr_employee_birth_name/models/hr_employee.py]: Why not calling this field simply `birth_name` ?

Like in https://github.com/OCA/partner-contact/tree/7.0/partner_birth_name

**Review by yvaucher — CHANGES_REQUESTED** [review]

File: hr_employee_birth_name/README.rst (hunk 3 above)
Comment (yvaucher) [reply hr_employee_birth_name/README.rst]: Please have a look at https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: hr_employee_birth_name/models/hr_employee.py (hunk 4 above)
Comment (leemannd) [reply hr_employee_birth_name/models/hr_employee.py]: @yvaucher Didn't saw it. It has been removed since V8. Do you know why?

Comment (yvaucher) [reply hr_employee_birth_name/models/hr_employee.py]: Noone did the migration. But it's okay the other is on partner and this one is on hr_employee

**Comment by leemannd** [comment]
@yvaucher Updated

**Comment by yvaucher** [comment]
@leemannd what about the field name ?

**Comment by leemannd** [comment]
Didn't understood well your comment. Doing the change

**Review by gurneyalex — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Review by yvaucher — APPROVED** [review]

**Pull request merged**